### PR TITLE
web/css/charset zh-CN fixup

### PR DESCRIPTION
**Pull request by jasonren0403**
Fix `Error: could not find syntax for this item` syntax displayed on current page

### CODE REVIEW

**Comment by jasonren0403** [comment]
`check redirects` has errored with `Errors loading redirects`, maybe it's an upstream issue?

File: files/zh-cn/web/css/@charset/index.html [hunk 1]
```diff
@@ -11,47 +15,46 @@
...
+<p> <strong><code>@charset </code></strong><a href="/zh-CN/docs/Web/CSS">CSS</a> <a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a>  指定样式表中使用的字符编码。它必须是样式表中的第一个元素，而前面不得有任何字符。因为它不是一个<a href="https://developer.mozilla.org/zh-CN/docs/Web/CSS/Syntax#css_语句">嵌套语句</a>，所以不能在<a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a><a href="https://developer.mozilla.org/zh-CN/docs/Web/CSS/At-rule#条件规则组">条件组</a>中使用。如果有多个 <strong><code>@charset </code></strong><a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a>被声明，只有第一个会被使用，而且不能在 HTML 元素或 HTML 页面的字符集相关 {{ HTMLElement("style") }} 元素内的样式属性内使用。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/@charset/index.html:18]:
```suggestion
<p><strong><code>@charset</code></strong> <a href="/zh-CN/docs/Web/CSS">CSS</a> <a href="/zh-CN/docs/CSS/At-rule">@规则</a>指定样式表中使用的字符编码。它必须是样式表中的第一个元素，而前面不得有任何字符。因为它不是一个<a href="/zh-CN/docs/Web/CSS/Syntax#css_语句">嵌套语句</a>，所以不能在<a href="/zh-CN/docs/CSS/At-rule">@规则</a><a href="/zh-CN/docs/Web/CSS/At-rule#条件规则组">条件组</a>中使用。如果有多个 <strong><code>@charset</code></strong> <a href="/zh-CN/docs/CSS/At-rule">@规则</a>被声明，只有第一个会被使用，而且不能在 HTML 元素或 HTML 页面的字符集相关 {{ HTMLElement("style") }} 元素内的样式属性内使用。</p>
```

File: files/zh-cn/web/css/@charset/index.html [hunk 2]
```diff
@@ -60,4 +63,11 @@ <h2 id="规范">规范</h2>
...
 <h2 id="Browser_compatibility">浏览器兼容性</h2>
 
-<p>{{Compat("css.at-rules.charset")}}</p>
+<p>{{Compat}}</p>
+
+<h2 id="参见">参见</h2>
+
+<ul>
+<li><a href="https://developer.mozilla.org/zh-CN/docs/Glossary/character_set">Character Set</a>术语条目</li>
+<li><a href="https://developer.mozilla.org/zh-CN/docs/Glossary/Unicode">Unicode</a>术语条目</li>
```
Comment (yin1999) [inline files/zh-cn/web/css/@charset/index.html:72]:
```suggestion
<li><a href="/zh-CN/docs/Glossary/character_set">字符集</a>术语条目</li>
<li><a href="/zh-CN/docs/Glossary/Unicode">Unicode</a> 术语条目</li>
```

File: files/zh-cn/web/css/@charset/index.html [hunk 3]
```diff
@@ -11,47 +15,46 @@
...
-<pre>@charset "UTF-8";
+<pre class="brush: css">@charset "UTF-8";
 @charset "iso-8859-15";
 </pre>
 
-<p>where:</p>
+<h3 id="语法格式">语法格式</h3>
+
+<pre class="brush: css">
+@charset "<charset>";
```
Comment (yin1999) [inline files/zh-cn/web/css/@charset/index.html:41]:
```suggestion
@charset "&lt;charset&gt;";
```

File: files/zh-cn/web/css/@charset/index.html [hunk 4]
```diff
@@ -11,47 +15,46 @@
...
+<p> <strong><code>@charset </code></strong><a href="/zh-CN/docs/Web/CSS">CSS</a> <a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a>  指定样式表中使用的字符编码。它必须是样式表中的第一个元素，而前面不得有任何字符。因为它不是一个<a href="https://developer.mozilla.org/zh-CN/docs/Web/CSS/Syntax#css_语句">嵌套语句</a>，所以不能在<a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a><a href="https://developer.mozilla.org/zh-CN/docs/Web/CSS/At-rule#条件规则组">条件组</a>中使用。如果有多个 <strong><code>@charset </code></strong><a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a>被声明，只有第一个会被使用，而且不能在 HTML 元素或 HTML 页面的字符集相关 {{ HTMLElement("style") }} 元素内的样式属性内使用。</p>
 
 <p>此 <a href="https://developer.mozilla.org/zh-CN/docs/CSS/At-rule">@规则</a> 在某些 CSS 属性中使用非 ASCII 字符时非常有用，例如 {{ cssxref("content") }}。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/@charset/index.html:20]:
```suggestion
<p>此 <a href="/zh-CN/docs/CSS/At-rule">@规则</a>在某些 CSS 属性中使用非 ASCII 字符时非常有用，例如 {{ cssxref("content") }}。</p>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E7%AB%99%E5%86%85%E9%93%BE%E6%8E%A5%E7%BF%BB%E8%AF%91

**Pull request merged**